src/OrchardCore.Modules/OrchardCore.Search.Lucene/Migrations.cs

```diff
@@ -174,28 +174,31 @@ public int Create()
                             {
                                 logger.LogDebug("Updating Lucene indices settings and queries");
                             }
+                            var quoteTableName = dialect.QuoteForTableName(table, session.Store.Configuration.Schema);
+                            var quoteContentColName = dialect.QuoteForColumnName("Content");
+                            var quoteTypeColName = dialect.QuoteForColumnName("Type");
```

Comment:
I suggest `var quotedTableName`, `quotedContentColumnName` and `quotedTypeColumnName`.